packages/assets-controllers/src/AccountTrackerController.test.ts

```diff
@@ -499,17 +561,18 @@ describe('AccountTrackerController', () => {
 
   it('should call refresh every interval on legacy polling', async () => {
     const poll = sinon.spy(AccountTrackerController.prototype, 'poll');
-    const controller = new AccountTrackerController(
-      {
-        onPreferencesStateChange: jest.fn(),
-        getIdentities: () => ({}),
-        getSelectedAddress: () => '',
+    const { controller } = setupController({
+      options: {
         getMultiAccountBalancesEnabled: () => true,
         getCurrentChainId: () => '0x1',
         getNetworkClientById: jest.fn(),
       },
-      { provider, interval: 100 },
-    );
+      config: { provider, interval: 100 },
+      mocks: {
+        selectedAccount: ACCOUNT_1,
```

Comment:
This used to be `getSelectedAddress: () => ''` so like, none account were selected